extension/js/common/browser.ts

```diff
@@ -214,6 +214,14 @@ export class Env {
     return ['gmail', 'inbox']; // async because storage may be involved in the future
   }
 
+  public static getBaseUrl = () => {
```

Comment:
Pulled URL-getting logic into Env for convenience and code cleanliness.

Reply:
absolutely!